### PR TITLE
Fix cancer_rank_options typo in case report

### LINKED ISSUE #4554: jinja2.exceptions.UndefinedError: 'cancer_rank_options' is undefined
```
2024-04-08 14:11:22,266 - [scout.server.app](http://scout.server.app/) - ERROR: Exception on /cust000/F0045564-RNAreanalys/case_report [GET] [in /venv/lib/python3.8/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/venv/lib/python3.8/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/worker/app/scout/server/utils.py", line 92, in decorated_function
    return render_template(template_name, **context)
  File "/venv/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(app, template, context)
  File "/venv/lib/python3.8/site-packages/flask/templating.py", line 130, in _render
    rv = template.render(context)
  File "/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/venv/lib/python3.8/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/worker/app/scout/server/blueprints/cases/templates/cases/case_report.html", line 1439, in top-level template code
    <span class="badge badge-pill badge-secondary" title="Manual rank">{{ manual_rank_options[variant.manual_rank]['label'] }}</span>
  File "/home/worker/app/scout/server/templates/report_base.html", line 19, in top-level template code
    {% block body %}
  File "/home/worker/app/scout/server/blueprints/cases/templates/cases/case_report.html", line 63, in block 'body'
    {{ tagged_panel() }}
  File "/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 777, in _invoke
    rv = self._func(*arguments)
  File "/home/worker/app/scout/server/blueprints/cases/templates/cases/case_report.html", line 419, in template
    {{ variant_content(variant, loop.index) }}
  File "/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 777, in _invoke
    rv = self._func(*arguments)
  File "/home/worker/app/scout/server/blueprints/cases/templates/cases/case_report.html", line 444, in template
    {{ sv_variant_content(variant, index) }}
  File "/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 777, in _invoke
    rv = self._func(*arguments)
  File "/home/worker/app/scout/server/blueprints/cases/templates/cases/case_report.html", line 1030, in template
    <span class="badge badge-pill badge-secondary" title="Tier">{{ cancer_rank_options[variant.cancer_tier]['label'] }}</span>
  File "/venv/lib/python3.8/site-packages/jinja2/environment.py", line 466, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'cancer_rank_options' is undefined
```

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Fix #4554 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Locally, pick the RNA case and add cancer tier to a variant
2. Go to case report

**Expected outcome**:
- Main branch will crash
- This branch will show the report

**Review:**
- [x] code approved by TB
- [x] tests executed by TB

### CODE REVIEW

**Comment by TereseBo** [comment]
On main branch, case_report can not load when a Tier has been set for a variant:

image

image

On this branch the case_report loads as expected and includes the cancer tier
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Cancer case_report panel-table no longer contains inheritance information
 - Case report pinned variants card now displays info text if all pinned variants are present in causatives
 - Darkmode setting now applies to the comment-box accordion
+- Typo in case report causing `cancer_rank_options is undefined` error
```
Comment (TereseBo) [inline CHANGELOG.md:16]: Perhaps state that it was fixed?

**Review by TereseBo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md:16]: It's already in the "Fixed" category, I think it's obvious that it's a fixed typo

Comment (northwestwitch) [reply CHANGELOG.md:16]: Thanks for the review, merging!

**Pull request merged**